balsamiq.md

```diff
@@ -0,0 +1,222 @@
+# Case Study - Balsamiq Docs: Static Doesn't Have to Be a Limitation
+
+Leon Barnard describes how the second version of the Balsamiq Docs static site went beyond purely functional to do things the team didn't think Markdown and static sites could do.
```

Comment:
Add your title, if you don't mind, I think it sets great context for the article. Such as, "Leon Barnard, Designer and Write at Balsamiq, describes how the second version of the ..." Or, maybe since you switch to first person (we, I wrote, I coded, and so on) you could also lead with "Here's a story about how the second version of a docs site went beyond purely function into new areas for a static site. I'm Leon Barnard, designer and writer at Balsamiq, here to describe how the second version of our Balsamiq Docs static site can now do things the team didn't think Markdown and static sites could do." Your choice, I do like the friendly voice and conversational tone throughout.